pkg/engine/lifecycletest/import_test.go

```diff
@@ -1261,3 +1263,85 @@ func TestImportInputDiff(t *testing.T) {
 	assert.Equal(t, upInputs, snap.Resources[1].Inputs)
 	assert.Equal(t, readOutputs, snap.Resources[1].Outputs)
 }
+
+func TestImportDefaultProvider(t *testing.T) {
```

Comment:
Quick doc comment about what it tests? I think the name alone is ambiguous since we are testing more than just import/default providers here.

Reply:
Sure